src/interactivity/win32/UiaTextRange.cpp

```diff
@@ -20,7 +20,7 @@ using namespace Microsoft::Console::Interactivity::Win32::UiaTextRangeTracing;
 //#define UIATEXTRANGE_DEBUG_MSGS 1
 #undef UIATEXTRANGE_DEBUG_MSGS
 
-IdType UiaTextRange::id = 0;

```

Comment:
I want @adiviness to look at this specifically. UIA is fraught with 0 vs 1 things.

Reply:
this should be fine, I believe the id is used only for tracelogging.